service/src/test/java/tds/session/web/endpoints/SessionControllerTest.java

```diff
@@ -73,32 +76,42 @@ public void shouldThrowNotFoundExceptionWhenSessionCannotBeFoundById() {
     @Test
     public void shouldPauseSession() {
         final UUID sessionId = UUID.randomUUID();
-        final String newStatus = "paused";
-        final Instant dateChanged = Instant.now();
-        Session session = new Session.Builder()
+        UUID browserKey = UUID.randomUUID();
+        long proctorId = 1L;
+        PauseSessionRequest request = new PauseSessionRequest(proctorId, browserKey);
+        Session mockClosedSession = new Session.Builder()
             .withId(sessionId)
-            .withStatus(newStatus)
-            .withDateChanged(dateChanged)
+            .withProctorId(proctorId)
+            .withBrowserKey(browserKey)
+            .withDateChanged(Instant.now())
+            .withDateEnd(Instant.now())
+            .withStatus("closed")
             .build();
+        PauseSessionResponse mockClosedResponse = new PauseSessionResponse(mockClosedSession);
 
-        PauseSessionResponse pauseSessionResponse = new PauseSessionResponse(session);
-
-        when(mockSessionService.pause(sessionId, newStatus)).thenReturn(Optional.of(pauseSessionResponse));
-        ResponseEntity<PauseSessionResponse> responseEntity = controller.pause(sessionId, newStatus);
-        verify(mockSessionService).pause(sessionId, newStatus);
+        when(mockSessionService.pause(sessionId, request)).thenReturn(new Response<>(mockClosedResponse, new ValidationError[]{}));
+        ResponseEntity<Response<PauseSessionResponse>> responseEntity = controller.pause(sessionId, request);
+        verify(mockSessionService).pause(sessionId, request);
 
         assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
-        assertThat(responseEntity.getBody().getExamIds()).isEqualTo(pauseSessionResponse.getExamIds());
-        assertThat(responseEntity.getBody().getStatus()).isEqualTo(newStatus);
-        assertThat(responseEntity.getBody().getDateChanged()).isEqualTo(dateChanged);
+        PauseSessionResponse resultFromService = responseEntity.getBody().getData().get();
+        assertThat(resultFromService.getSessionId()).isEqualTo(sessionId);
+        assertThat(resultFromService.getStatus()).isEqualTo("closed");
+        assertThat(resultFromService.getDateEnded().getMillis()).isEqualTo(mockClosedResponse.getDateEnded().getMillis());
```

Comment:
I assume you checked but do you have to do the `getMillis()` call?  I think `Instant.equals(Instant)` should work.